src/components/Navbar.js

```diff
@@ -3,13 +3,13 @@ import { Link } from 'react-router-dom';
 
 function Navbar() {
   return (
-    <nav>
-      <div>Bookstore CMS</div>
+    <nav className="navbar">
+      <div className="logo">Bookstore CMS</div>
       <ul>
```

Comment:
- [ ] Check this out,
<img width="1235" alt="1" src="https://github.com/wolterbwambale/bookstore-app/assets/77843050/eab62356-d1d8-4f02-9ead-064f07f1c14c">

1. Your book is missing some alignment on the top, bottom, and left as shown and compared to the original template
2. The bg-color of your design is `gray` while on the original design, it is `white`
3. Your book is missing a small border-radius property as shown and compared to the original design